ci/pipelines/e2e-helpers.yml

```diff
@@ -53,7 +53,7 @@ resources:
     icon: github
     source:
       uri: https://github.com/alphagov/pay-ci
-      branch: master
+      branch: PP-10335-send-e2e-helpers-job-metric-to-hosted-graphite
```

Comment:
One last thing to do before we merge this - change this back to `master` 😸

Reply:
Done :)